packages/@tinacms/cli/src/cmds/compile/index.ts

```diff
@@ -39,76 +42,119 @@ export const resetGeneratedFolder = async () => {
   await fs.outputFile(path.join(tinaGeneratedPath, '.gitignore'), 'db')
 }
 
+const cleanup = async ({
+  tinaConfigPath,
+  tinaTempPath,
+}: {
+  tinaConfigPath: string
+  tinaTempPath: string
+}) => {
+  // Delete the node require cache for .tina temp folder
+  Object.keys(require.cache).map((key) => {
+    if (key.startsWith(tinaTempPath)) {
+      delete require.cache[require.resolve(key)]
+    }
+  })
+  try {
+    const schemaFunc = require(path.join(tinaTempPath, 'schema.js'))
+    const schemaObject: TinaCloudSchema = schemaFunc.default
+    await fs.outputFile(
+      path.join(tinaConfigPath, 'schema.json'),
+      JSON.stringify(schemaObject, null, 2)
+    )
+    await fs.remove(tinaTempPath)
```

Comment:
this line will still get skipped if anything throws from this `try` block, and we probably want to make sure it runs regardless

Reply:
Aw yes, I overlooked what the cleanup function was doing. Should be working as expected now.